1-js/02-first-steps/04-variables/article.md

```diff
@@ -1,343 +1,348 @@
 
 ```js run no-strict
-// note: no "use strict" in this example
+// नोट: इस उदाहरण में कोई `use strict` नहीं है
 
-num = 5; // the variable "num" is created if it didn't exist
+num = 5; // variable "num" बनाया जाता है अगर यह मौजूद नहीं था
 
 alert(num); // 5
 ```
 
-This is a bad practice and would cause an error in strict mode:
+यह एक बुरा अभ्यास है और इससे strict मोड में त्रुटि होगी:
 
 ```js
 "use strict";
 
 *!*
-num = 5; // error: num is not defined
+num = 5; // त्रुटि: num को परिभाषित नहीं किया गया है
 */!*
 ```
 ````
 
 ## Constants
 
-To declare a constant (unchanging) variable, use `const` instead of `let`:
+एक constant (अपरिवर्तनीय) variable घोषित करने के लिए, `let` के बजाय `const` का उपयोग करें:
 
 ```js
 const myBirthday = '18.04.1982';
 ```
 
 Variables declared using `const` are called "constants". They cannot be reassigned. An attempt to do so would cause an error:
+`Const` का उपयोग करके घोषित variables को "constants" कहा जाता है। उन्हें आश्वस्त नहीं किया जा सकता है। ऐसा करने का प्रयास त्रुटि पैदा करेगा:
 
 ```js run
 const myBirthday = '18.04.1982';
 
-myBirthday = '01.01.2001'; // error, can't reassign the constant!
+myBirthday = '01.01.2001'; // त्रुटि, constant आश्वस्त नहीं कर सकता!
 ```
 
-When a programmer is sure that a variable will never change, they can declare it with `const` to guarantee and clearly communicate that fact to everyone.
+जब एक प्रोग्रामर यह सुनिश्चित करता है कि एक variable कभी नहीं बदलेगा, तो वे गारंटी देने के लिए इसे `const` के साथ घोषित कर सकते हैं और उस तथ्य को सभी को स्पष्ट रूप से बता सकते हैं।
 
 
-### Uppercase constants
+### बड़े अक्षर वाले constants
 
-There is a widespread practice to use constants as aliases for difficult-to-remember values that are known prior to execution.
+मुश्किल से याद रखने वाले मूल्यों के लिए constants का उपयोग करना एक व्यापक अभ्यास है जिसे क्रियान्वयन से पहले जाना जाता है।
 
-Such constants are named using capital letters and underscores.
+ऐसे constants को बड़े अक्षरों और अंडरस्कोर का उपयोग करके नाम दिया गया है।
 
-For instance, let's make constants for colors in so-called "web" (hexadecimal) format:
+उदाहरण के लिए, चलो तथाकथित "वेब" (हेक्साडेसिमल) प्रारूप में रंगों के लिए constants बनाते हैं:
 
 ```js run
 const COLOR_RED = "#F00";
 const COLOR_GREEN = "#0F0";
 const COLOR_BLUE = "#00F";
 const COLOR_ORANGE = "#FF7F00";
 
-// ...when we need to pick a color
+// ...जब हमें एक रंग चुनने की आवश्यकता होती है
 let color = COLOR_ORANGE;
 alert(color); // #FF7F00
 ```
 
-Benefits:
+लाभ:
 
-- `COLOR_ORANGE` is much easier to remember than `"#FF7F00"`.
-- It is much easier to mistype `"#FF7F00"` than `COLOR_ORANGE`.
-- When reading the code, `COLOR_ORANGE` is much more meaningful than `#FF7F00`.
+-`COLOR_ORANGE` को `"#FF7F00"` की तुलना में याद रखना बहुत आसान है।
+- `COLOR_ORANGE` की तुलना में `#FF7F00` को गलत ठहराना ज्यादा आसान है।
+- कोड पढ़ते समय `COLOR_ORANGE`, `#FF7F00` की तुलना में बहुत अधिक सार्थक है।
```

Comment:
सार्थक does not seem to be a perfect fit here

Reply:
Done